dbt_subprojects/tokens/models/prices/sonic/prices_sonic_tokens.sql

```diff
@@ -0,0 +1,26 @@
+{% set blockchain = 'sonic' %}
+
+{{ config(
+    schema = 'prices_' + blockchain,
+    alias = 'tokens',
+    materialized = 'table',
+    file_format = 'delta',
+    tags = ['static']
+    )
+}}
+
+SELECT
+    token_id
+    , '{{ blockchain }}' as blockchain
+    , symbol
+    , contract_address
+    , decimals
+FROM
+(
+    VALUES
+    ('ftm-fantom', 'wS', 0x039e2fB66102314Ce7b64Ce5Ce3E5183bc94aD38, 18) -- looks like FTM <--> S token are same price and 1:1 upgrade according to docs
```

Comment:
using FTM for now, since it appears the prices match + there is 1:1 upgrade via their docs. would be nice to update if/when coinpaprika has sonic available

Reply:
this is by far most transferred token though, i think it's important we include